src/app/garfile/departure/index.njk

```diff
@@ -124,19 +124,18 @@
 
         {# Start departure port #}
         <div class="govuk-form-group {{ m.form_group_class('departurePort', errors) }}">
-          <label class="govuk-label govuk-label--m" for="departurePort">{{__('field_departure_port')}}</label>
+          <label id="departurePort-label" class="govuk-label govuk-label--m" for="departurePort">{{__('field_departure_port')}}</label>
           <span id="departurePort-hint" class="govuk-hint">{{__('field_port_hint', "YYYY")}}</span>
           {{ m.error_message('departurePort', errors, __) }}
-          <select class="govuk-select" id="departurePort" name="departurePort"
-          value="{{ cookie.getGarDepartureVoyage().departurePort | upper }}">
+          <select id="departurePort" name="departurePort" class="govuk-select" 
+            value="{{ cookie.getGarDepartureVoyage().departurePort | upper }}"
+            aria-labelledby="departurePort-label"
+            aria-describedby="departurePort-hint{{ ' departurePort-error' if errors | containsError('departurePort') }}">
```

Comment:
Hopefully this is sufficient to work with the accessible autocomplete in regards screenreader functionality.